src/AzureDeprecation.Runtimes.AzureFunctions/ValueStopwatch.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Diagnostics;
+
+namespace AzureDeprecation.Runtimes.AzureFunctions
+{
+    public struct ValueStopwatch

```

Comment:
What is this used for? I didn't see it so must have looked over it

Reply:
Shamelessly copied (and extended for the milliseconds) from here: https://github.com/dotnet/aspnetcore/blob/main/src/Shared/ValueStopwatch/ValueStopwatch.cs